Sources/CombineCoreBluetooth/Models/L2CAPChannel.swift

```diff
@@ -1,6 +1,9 @@
 import Foundation
+import CoreBluetooth
 
 public struct L2CAPChannel {
+  // Need to keep a reference to this so the system doesn't close the channel
+  let rawValue: CBL2CAPChannel
```

Comment:
Could this be optional, so that if someone wanted to create a mock channel for testing, they wouldn't be forced to make a core bluetooth type that may not be creatable or customizable in a test?
```suggestion
  let rawValue: CBL2CAPChannel?
```

Reply:
gotcha, thank you!